lib/workers/types.ts

```diff
@@ -115,4 +115,5 @@ export interface BranchConfig
   packageFiles?: Record<string, PackageFile[]>;
   prBlockedBy?: PrBlockedBy;
   prNo?: number;
+  stopUpdatingLabelPresent?: boolean;
```

Comment:
```suggestion
  stopUpdating?: boolean;
```

Let's shorten this (requires some other code changes too in other files)